packages/nextjs/services/database/config/schema.ts

```diff
@@ -40,6 +46,8 @@ export const users = pgTable(
     socialInstagram: varchar({ length: 255 }),
     socialDiscord: varchar({ length: 255 }),
     socialEmail: varchar({ length: 255 }),
+    batchId: integer().references(() => batches.id),
+    batchStatus: batchUserStatusEnum(),
```

Comment:
As I mentioned in the PR description, 1:1 relation

Reply:
I think we also need to add the relations: 

```ts
export const usersRelations = relations(users, ({ many, one }) => ({
  userChallenges: many(userChallenges),
  batch: one(batches, {
    fields: [users.batchId],
    references: [batches.id],
  }),
}));

export const batchesRelations = relations(batches, ({ one }) => ({
  users: one(users, {
    fields: [batches.id],
    references: [users.batchId],
  }),
}));
```
This will make sure when we query get proper types, if we don't add then it errors out while querying 🙌

![image](https://github.com/user-attachments/assets/7f792a5a-6563-4e22-9a8d-a1bd46e7fc90)

Reply:
Oh, I omitted that because I thought those were for 1:N / N:N relations, but now I see that it's for every relation! And it's only for types, etc (migration files are still the same!)

Thank Shiv! 336a8e6d1f58